tests/jvm/src/test/resources/metac.owners

```diff
@@ -0,0 +1,206 @@
+_empty_.Empty# => _empty_.
```

Comment:
I'm still sceptical about the value of owners since I believe we can compute this information directly from a prefix tree with children/owner links. But I can see the value in not having to rely on sophisticated data structures

Reply:
The idea that I want to try is to treat symbols as completely opaque tokens, with everything that we need to know living in `SymbolInformation`. I know that this is not how we've been rolling so far, but I think it's an interesting goal to have. It also simplifies the API.